drivers/console/rtt_console.c

```diff
@@ -18,14 +18,63 @@
 extern void __printk_hook_install(int (*fn)(int));
 extern void __stdout_hook_install(int (*fn)(int));
 
+static bool host_present;
+
+/** @brief Wait for fixed period.
+ *
+ */
+static void wait(void)
+{
+	if (k_is_in_isr()) {
+		k_busy_wait(1000*CONFIG_RTT_TX_RETRY_DELAY_MS);
```

Comment:
spaces: `1000 * CONFIG_RTT_TX_RETRY_DELAY_MS`

Reply:
Should we really block if we are in an ISR by the way? I wonder if this should be configurable. I guess this will be a moot point once we used the packed stream that comes with the new logging subsystem, but I would still make this configurable so that we don't start getting complaints from users about ISRs running for a long time.

Reply:
added option in kconfig